internal/gengapic/client_init_test.go

```diff
@@ -549,7 +546,12 @@ func TestClientInit(t *testing.T) {
 				Parameter: tst.parameter,
 				ProtoFile: fds,
 			}
-			g.init(&request)
+			g, err := newGenerator(&request)
+			if err != nil {
+				t.Fatal(err)
+			}
+			g.apiName = "Awesome Foo API"
+			g.imports = map[pbinfo.ImportSpec]bool{}
```

Comment:
Is this redundant given the init of this field in `newGenerator` (line 97)?

Reply:
removed